dead-links.json

```diff
@@ -0,0 +1,24 @@
+{
+  "total": 260,
+  "404": {
+    "urls": [
+      "https://stackexchange.com/users/?tab\u003dinbox",
+      "https://stackexchange.com/users/?tab\u003dreputation",
+      "https://stackoverflow.com/posts/878348/edit",
+      "https://stackoverflow.com/posts/878362/edit",
+      "https://stackoverflow.com/posts/878559/edit",
+      "https://stackoverflow.com/posts/878496/edit",
+      "https://stackoverflow.com/posts/19183119/edit",
+      "https://stackoverflow.com/questions/878348/where-do-i-configure-log4j-in-a-junit-test-class?lastactivity"
+    ],
+    "size": 8
+  },
+  "dead": 9,
+  "50x": {
+    "urls": [
+      "https://linkedin.com/company/stack-overflow"
+    ],
+    "size": 1
+  },
+  "url": "https://stackoverflow.com/questions/878348/where-do-i-configure-log4j-in-a-junit-test-class"
+}
```

Comment:
it's a good practice to add the newline character, so that each file in repository (besides binary files) ends with an empty line